UnitsNet.Tests/CustomCode/EntropyTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated (once) by \generate-code.bat, but will not be
+//     regenerated when it already exists. The purpose of creating this file is to make 
+//     it easier to remember to implement all the unit conversion test cases.
+// 
+//     Whenever a new unit is added to this quantity and \generate-code.bat is run,
+//     the base test class will get a new abstract property and cause a compile error
+//     in this derived class, reminding the developer to implement the test case
+//     for the new unit.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyUnit.extra.cs files to add code to generated quantities.
+//     Add Extensions\MyUnitExtensions.cs to decorate quantities with new behavior.
+//     Add UnitDefinitions\MyUnit.json and run GeneratUnits.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Copyright (c) 2007 Andreas Gullberg Larsen (angularsen@gmail.com).
+// https://github.com/angularsen/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class EntropyTests : EntropyTestsBase
+    {
+        protected override double CaloriesPerKelvinInOneJoulePerKelvin => 1 / 4.184;
```

Comment:
Test constants should not be computations, but rather constants looked up in some reference material. This is an important step to avoid copying the same implementation over to the test. The values look correct though.

F.ex use this converter: http://www.gordonengland.co.uk/conversion/entropy.htm

Reply:
Well I think using only one constant and describing manipulations for conversions is better. What do I mean:

1 Calorie equals to 4.184 Joule, but if we start from the other end 1 Joule equals to 0.239006 Calorie. Conversion constant is the same (by value) but reader can't say that they are the same until calculating it.

However if we pick 4.184 or 0.239006 as main constant, we can describe other conversions as manipulations of the main constant and reader can easily verify that they are the same thus avoiding conversion bugs.

For example:
1 Calorie equals to 4.184 Joules and 1 Joule equals to 1/4.184.


But anyway, I'll refactor as you wish.

Reply:
So we should show computation steps in implementations, resulting values in tests, right?

For example: 1/4.184 in implementation, 0.239006 in test.

Reply:
We can implement Specific Entropy. And you already merged Molar Entropy :)

Reply:
I'm going to work on other units you suggested.

Reply:
Correct, the important part is to basically double check that the implementation is correct. You can reuse the constant and multiple it by `1e3` for the kilo prefixes etc. That's fine I think and is easy to confirm for correctness.